vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/Upload.java

```diff
@@ -565,6 +568,19 @@ public Registration addFileRejectedListener(
         return addListener(FileRejectedEvent.class, listener);
     }
 
+    /**
+     * Adds a listener for {@code file-remove} events fired when a file is
```

Comment:
nit: I think this technical detail regarding the web component `file-remove` event is unnecessary in the Java API.

Reply:
Removed the mention of the event name.